src/App.vue

```diff
@@ -23,19 +23,16 @@ export default {
       localStorage.setItem('memos', JSON.stringify(this.memos))
       this.$router.push('/')
     },
-    createMemo (e) {
-      const title = e.split('\n')[0]
+    createMemo (newContent) {
```

Comment:
新しいcontentかどうかはこのメソッドにとって大きな問題ではないので、`content`だけで良さそうです。updateも同様。

```suggestion
    createMemo (content) {
```